### PR TITLE
Market/get agreement caller

### PR DESCRIPTION
**Pull request by tworec**
contains also #914

### CODE REVIEW

**Review by Wiezzel — COMMENTED** [review]
Please explain what is the reason for adding `caller_id` to `get_agreement`. This is a local endpoint so I don't see a reason for checking who called it/

File: core/market/src/market/agreement.rs [hunk 1]
```diff
@@ -10,6 +10,7 @@ use ya_service_bus::typed::ServiceBinder;
 
 use crate::db::dao::AgreementDao;
 use crate::db::model::{AgreementId, Owner};
+use ya_client::model::ParseError;
```
Comment (nieznanysprawiciel) [inline core/market/src/market/agreement.rs:13]: This use is grouped together with crate uses. But it isn't important at all

File: core/market/src/negotiation/common.rs [hunk 2]
```diff
@@ -589,13 +574,6 @@ impl CommonBroker {
         Ok(())
     }
 
-    pub(crate) fn parse_caller(caller: &str) -> Result<NodeId, CallerParseError> {
-        NodeId::from_str(caller).map_err(|e| CallerParseError {
-            caller: caller.to_string(),
-            e: e.to_string(),
-        })
-    }
-
```
Comment (nieznanysprawiciel) [inline core/market/src/negotiation/common.rs:598]: 👍🏻

File: Cargo.toml [hunk 3]
```diff
@@ -172,8 +172,8 @@ ya-sb-router = { path = "service-bus/router" }
 ya-sb-util = { path = "service-bus/util" }
 
 ## CLIENT
-ya-client = { git = "https://github.com/golemfactory/ya-client.git", rev = "87f106498a7a7b1519d82d0981fc21b260a89cf2"}
-ya-client-model = { git = "https://github.com/golemfactory/ya-client.git", rev = "87f106498a7a7b1519d82d0981fc21b260a89cf2"}
+ya-client = { git = "https://github.com/golemfactory/ya-client.git", rev = "6381a56024acc197ac7135aaa85bf4e12155f647"}
+ya-client-model = { git = "https://github.com/golemfactory/ya-client.git", rev = "6381a56024acc197ac7135aaa85bf4e12155f647"}
```
Comment (nieznanysprawiciel) [inline Cargo.toml:176]: Remember to switch to commit on master branch after merging https://github.com/golemfactory/ya-client/pull/93

**Review by nieznanysprawiciel — APPROVED** [review]
Approve for market part.

**Comment by mfranciszkiewicz** [comment]
stale changeset; @tworec please re-create the PR if you believe this is still relevant

**Pull request closed without merging**